README.md

```diff
@@ -85,11 +92,11 @@ yarn
 cd ios && pod install
 ```
 
-4. It is required to have `WebRTC.framework` M112 builds. You can find it in `libWebRTC.zip` inside the `ios` folder.
+1. It is required to have `WebRTC.framework` M112 builds. You can find it in `ios/libWebRTC.zip`.
```

Comment:
This should still be 4.

Reply:
You're right. I don't know why that changed.